src/effects/Effect.cpp

```diff
@@ -2387,8 +2385,6 @@ void Effect::Preview(bool dryOnly)
 
       mixLeft->Offset(-mixLeft->GetStartTime());
       mixLeft->SetSelected(true);
-      WaveTrackView::Get( *mixLeft )
```

Comment:
We remove `WaveTrackView` from `Preview`, but should implement this functionality (optionaly preview) like standalone for possibilty user on/off in preferences and using it in various Effects implementations (Effects, NyqiestEffect)?